src/v/crypto/tests/crypto_bench.cc

```diff
@@ -17,129 +17,8 @@
 #include <seastar/core/sleep.hh>
 #include <seastar/testing/perf_tests.hh>
 
-#include <gnutls/crypto.h>
-#include <gnutls/gnutls.h>
-
 static constexpr size_t inner_iters = 1000;
 
-template<gnutls_mac_algorithm_t Algo, size_t DigestSize>
-class hmac {
-    static_assert(DigestSize > 0, "digest cannot be zero length");
```

Comment:
Question about this removal, mostly for my own benefit - so in effect this validation is already done to our satisfaction and we can remove this? Or there's more work to do there and we'll build off an old commit?

Reply:
I'm good with the bench results.  Just left it in until we fully pulled gnutls

Reply:
Cool, yeah. I just haven't been following along I guess 😕